test.py

```diff
@@ -0,0 +1,61 @@
+import requests
+import time
+import csv
+
+def take_posts():
+    token = '6563ec986563ec986563ec98b7651856c7665636563ec980755cea2b6fdc8d85f0a81f2'
+    version = 5.131
+    domain = 'ruoil'
+    count = 100
+    offset = 0
+    all_posts = []
+    all_comments = []
+
+    while offset<1000:
+        response = requests.get('https://api.vk.com/method/wall.get',
+                                params={
+                                    'access_token': token,
+                                    'v': version,
+                                    'domain': domain,
+                                    'count': count,
+                                    'offset': offset
+                                }
```

Comment:
Смотри, в чем идея.
Делаешь сверху, вне цикла,
```
base_params = {
    'access_token': token,
    'v': version,
}
```

а тут - 
```suggestion
                                params=base_params | {
                                    'domain': domain,
                                    'count': count,
                                    'offset': offset
                                }
```

Главный профит в том, что ты можешь base_params использовать ниже, когда ты делаешь гет https://api.vk.com/method/wall.getComments